tests/unit/button/button_options.js

```diff
@@ -25,135 +25,80 @@ test( "disabled, explicit value", function() {
 	deepEqual( element.prop( "disabled" ), true, "element is not disabled" );
```

Comment:
This message is also misleading, like the equivalent above ("element is disabled"). Mixing positive and negative assertion messages is bad, we should stick with positive messages, describing the intended behaviour.